giraffe/src/components/StaticLegendBox.tsx

```diff
@@ -90,6 +91,10 @@ export const StaticLegendBox: FunctionComponent<StaticLegendBoxProps> = props =>
       headerTextMetrics,
       sampleTextMetrics,
     })
+    /* run this hook only for the height changes because the purpose is to set the height for user
+     * therefore, the dependencies should not include anything that doesn't affect height
+     */
+    // eslint-disable-next-line
```

Comment:
i think it's probably better from a code correctness and documentation standpoint to disable the specific rule, rather than the whole line.

Reply:
Yeah the problem is that without the plugin installed, disabling a specific rule that doesn't exist is in itself a lint error

Reply:
Since the original problem is a warning and not an error, maybe we should hold off on this until the plugin is actually installed